test/integration/stats_integration_test.cc

```diff
@@ -319,6 +320,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithRealSymbolTable) {
   // 2019/12/10  8779     35053       35000   use var-length coding for name lengths
   // 2020/01/07  9069     35548       35700   upstream: Implement retry concurrency budgets
   // 2020/01/07  9564     35580       36000   RefcountPtr for CentralCache.
+  // 2020/01/09  8889     35612       35740   api: add UpstreamHttpProtocolOptions message
```

Comment:
We should probably just leave the upper bound at 36000 but I don't feel strongly about it. cc @jmarantz